src/borg/borg-fight-defend.c

```diff
@@ -1426,12 +1470,16 @@ static int borg_defend_aux_smite_evil(int p1)
     if (!borg_spell_okay_fail(SMITE_EVIL, fail_allowed))
         return (0);
 
-    // !FIX !TODO !AJG we should probably figure out if we are about to fight
-    // something evil.
+    // if the borg is not about to fight something evil.
+    if (!near_evil())
+        return 0;
 
-    /* if we are in some danger but not much, go for a quick bless */
-    if ((p1 > avoidance * 1 / 10 && p1 < avoidance * 5 / 10)
-        || (borg_fighting_unique && p1 < avoidance * 7 / 10)) {
+    /* if we are in some danger but not much, go for a quick smite */
+    /* "some danger" defined as "10% of x and not more than 50% of x */
+    /* (not more than 70% when fighting a unique) */
+    /* where x is the danger we are avoiding, usually current hp */
+    if ((p1 > avoidance / 10 && (p1 < (avoidance * 5) / 10)
+        || (borg_fighting_unique && p1 < (avoidance * 7) / 10))) {
```

Comment:
In builds with Makefile.std, this triggers a warning suggesting parentheses (see https://github.com/angband/angband/actions/runs/8031683844/job/21940268967), use

```
if ((p1 > avoidance / 10 && p1 < (avoidance * 5) / 10)
        || (borg_fighting_unique && p1 < (avoidance * 7) / 10)) {
```

instead?  There's the similar test at line 1308 that should be changed to match this one (drop the "* 1" and wrap "avoidance * 5" in parentheses.

Reply:
ok mucked with the parens to get to what I think the code was trying to do initially.  I will watch for a warning

Reply:
I don't see any warnings for that now but lots of "redundant redeclaration".  I will take a look at those next.  I assume they weren't created by this checkin so I will look under a new branch